docker-compose.yml

```
@@ -0,0 +1,11 @@
+services:
+  todo_app:
+    image: todo_app:dev
+    env_file: .env
+    build: .
```

This works perfectly right now, but could easily break accidentally if somebody adds a new stage to your Dockerfile or swaps the stage order round, which can be unintuitive! I'd suggest therefore explicitly calling out the image target:
```suggestion
    build:
      context: .
      target: dev
```